docs/learn-treasury.md

```diff
@@ -52,9 +56,9 @@ The Treasury is funded from different sources:
 
 ## Creating a Treasury Proposal
 
-The proposer has to deposit 5% of the requested amount or 100 DOT (whichever is higher) as an
-anti-spam measure. This amount is burned if the proposal is rejected, or refunded otherwise. These
-values are subject to [governance](learn-governance) so they may change in the future.
+The proposer has to deposit 5% of the requested amount or 100 DOT / 3.3 KSM (whichever is higher) as
```

Comment:
Can the values for 100 DOT / 3.3 KSM be injecting in the injector script instead of writing them?